<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Blackman Lab - Research</title>
  <meta name="description" content="Blackman Lab - Research">
  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="/research/">
  <link rel="icon" href="/berkeley.png">



</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>
	
    <a class="navbar-brand" href="/">Blackman Lab &rarr; University of California, Berkeley</a>
	</div>
	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="/">Home</a></li>
		<li><a href="/team">Team</a></li>
		<li><a href="/vacancies">Vacancies</a></li>
		<li><a href="/publications">Publications</a></li>
		<li><a href="/research">Research</a></li>
		<li><a href="/pictures">Pictures</a></li>
	  </ul>
	</div>
  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        <div id="textid" class="col-sm-12">
  <h1 id="research">Research</h1>

<h2 id="sunflower-domestication">Sunflower Domestication</h2>
<h3 id="functional-studies-of-domestication-genes">Functional Studies of Domestication Genes</h3>
<p>Sunflower was domesticated ~4000 years ago by Native Americans in Eastern North America.  Through this process, it was transformed from a highly branched, many headed plant with small seeds into the unbranched crop plant with a single head containing large oilseeds.  Through evolutionary and functional studies, we have previously identified several candidate genes that contributed to this process.  Identifying domestication alleles has already led to novel insights into how gene duplications may contribute to the evolution of novelty (Blackman et al., 2010, see Gene Family Evolution).   The newly available sunflower genome will greatly expand our capacity to discover more of these domestication genes as well as understand more about sunflower’s complex history and the evolution of development in general.  Current projects are focused on determining the molecular and developmental functions of domestication alleles affecting sunflower life history and growth.</p>

<h3 id="tracing-the-assembly-of-a-domestication-syndrome-in-ancient-dna">Tracing the Assembly of A Domestication Syndrome In Ancient DNA</h3>
<p>Analyzing the distribution of domestication alleles in extant samples provides an entry point for resolving conflicting hypotheses about when and where plants species were first cultivated (e.g. Blackman et al., 2011). By harnessing the power of high throughput sequencing technologies, it is also now possible to obtain significant amounts of DNA sequence from archaeological samples.  Consequently, researchers can now observe changes in traits despite being unable to directly measure them on archaeological remains because it is possible to follow the frequencies of alleles known to contribute to these traits over historical time.</p>
<p>Sunflower has an excellent archaeological record over the past 4000 years, and our knowledge of the genes contributing to the origin of domesticated sunflower is ever growing.  Thus, in collaboration with Tom Gilbert and Nathan Wales at the University of Copenhagen, Bruce Smith and Robert Costello at the Smithsonian, and Kristen Gremillion at Ohio State University, we are beginning to mine the genomes of these samples (Wales et al. in press) with the ultimate goal of developing a timeline over which domestication traits arose and spread in North America.  </p>

<h2 id="adaptation-of-developmental-timing">Adaptation of Developmental Timing</h2>
<p>As species expand their ranges and colonize new areas, environmental conditions may change dramatically, selecting for the evolution of new developmental responses, alteration of existing responses, or elimination of plasticity altogether. We seek to understand how these processes have occurred historically to gain insight into the genetic architecture of adaptation and speciation, how plasticity may foster or hinder adaptation, and how gene networks are remodeled by evolution.</p>

<h3 id="photoperiod-response-adaptation-in-wild-sunflower">Photoperiod response adaptation in Wild Sunflower</h3>
<p>The common sunflower, Helianthus annuus, inhabits a broad geographical range over which the timing of the growth season varies greatly. Like many species, sunflower exhibits a latitudinal cline in flowering. Notably, it also shows transitions in the response of flowering to photoperiod along this gradient from insensitive to short-day sensitive to long-day sensitive.</p>
<p>In studying the genetic basis and ecological pressures driving this variation, we hope to understand whether natural selection on individual genes varies across environments due to plasticity in gene expression, changes in heritability or genetic architecture, and/or differences in the balance of selective agents in different environments. We have already identified gene expression correlates of this diversity (Blackman et al., 2011).  Current work is focused on mapping the loci involved (Henry et al. 2014) so that in the future we may also examine how the joint action of climatic and biotic selection pressures on these loci varies geographically in the field.</p>

<h3 id="genetics-of-threshold-trait-evolution-in-mimulus">Genetics of threshold trait evolution in mimulus</h3>
<p>Populations of the common monkeyflower, Mimulus guttatus, inhabit diverse habitats throughout western North America.  This species is an obligate long-day plant. That is, plants require a certain minimum day length for floral induction. Through surveys of flowering time diversity, we have found that this minimum threshold varies greatly within and between populations (Kooyers et al. 2015).  Notably, high elevation populations consistently require longer day lengths to induce flowering than nearby low elevation populations. Through genetic mapping and field studies, the lab is exploring the genetic basis and adaptive value of this variation. in collaboration with Daniel Runcie (University of California, Davis), Jason Sexton (University of California, Merced), and Akiko Satake (Kyushu University), we will also develop transcriptional network-informed models of flowering responses to predict population responses to current and future climates.</p>

<h2 id="gene-family-adaptation">Gene Family Adaptation</h2>
<h3 id="gene-duplications-and-the-origin-of-novelty">Gene duplications and the origin of Novelty</h3>
<p>Genes duplications contribute to the origin of novel traits in may ways (e.g. increased gene dosage, acquisition of new protein functions or expression domains, etc).  It has also been suggested nascent gene duplicates can lead to new functions through the evolution of novel competitive or cooperative interactions with each other, but empirical examples of such a process are lacking.</p>
<p>Our work on the <i>FLOWERING LOCUS T (FT)</i> gene family, which is critically involved in the control of flowering time and plant architecture, has identified a variant that allows us to test this hypothesis.  During sunflower domestication, selection nearly fixed an allele of one sunflower FT copy that causes this altered protein to interfere with the function of a second copy (Blackman et al., 2010).</p>
<p>Dramatic changes in which tissues and in what environments different FT copies are expressed have also evolved in within and between species in the sunflower lineage.  By characterizing the evolutionary timing and molecular mechanisms underlying this variation, our lab is working to gain additional insights into how gene duplications evolve new functions and contribute to diversity in phenotypic plasticity.</p>

<h2 id="solar-tracking">Solar Tracking</h2>

<p>Plants exploit a combination of internal and external cues to coordinate their daily investment in growth with daily fluctuations in resource availability. Solar tracking, or heliotropism, by developing heads is one of sunflower’s best known traits and is one excellent example of this form of developmental regulation (Vandenbrink et al. 2014).</p>
<p>Though solar tracking has captured the imagination of poets, gardeners, and botanists for decades, we have learned little about the genetic mechanisms that govern this growth behavior, its evolutionary origin, or its ecological function. In collaboration with Stacey Harmer at UC Davis, we are investigating these aspects of solar tracking using a combination of time-lapse imaging, field studies, quantitative genomics, gene expression, and mutant analysis (Atamian et al. 2016).</p>
<p><i>“The dials of these honeyed clocks<br />
Fringed with leaves of sun<br />
Turn obedient to the wheels<br />
That make the daystar run”<br />
– Robert P. Tristam Coffin
</i></p>


</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">
		  <p>&copy 2024 Blackman Lab. Site made with <a href="https://jekyllrb.com">Jekyll</a> 
			and adapted from the <a href="http://www.allanlab.org/aboutwebsite.html">Allan Lab's page.</a></p>
		</div>
		<div class="col-sm-4">
		  Funding:<br />
		  - <a href="https://berkeley.edu">PLACEHOLDER</a>
		</div>
		<div class="col-sm-4">
			University of California, Berkeley<br/>
			Department of Plant and Microbial Biology<br/>
			111 Koshland Hall #3102, Berkeley, CA 94720
            (<a href="https://maps.app.goo.gl/e9hHnQtJg7MDRP4x8">Maps</a>)
		</div>
	  </div>
	</div>
  </div>
</div>

<script
	src="https://code.jquery.com/jquery-2.2.4.min.js"
	integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44="
	crossorigin="anonymous"></script>
<script src="/js/bootstrap.min.js"></script>
<script
	  src="https://code.jquery.com/color/jquery.color-2.2.0.min.js"
	  integrity="sha256-aSe2ZC5QeunlL/w/7PsVKmV+fa0eDbmybn/ptsKHR6I="
	  crossorigin="anonymous">
</script>
<script type="text/javascript" src="/js/jquery.justifiedGallery.min.js"></script>
<script>
	$("#justified-gallery").justifiedGallery({
		  rowHeight : 200,
		  lastRow : 'nojustify',
		  margins : 3
	  });
</script>


  </body>

</html>
